Source/WebKit/gtk/tutorial.md

```diff
@@ -0,0 +1,382 @@
+# Create a Web Browser in GTK with WebKit
+
+This is a tutorial for creating a simple web browser with some basic functionalities in GTK with WebKit in C language.
+
+GTK is a multi-platform toolkit for creating graphical user interfaces; it contains a complete set of widgets to be reused and combined together in development.
+
+## Prerequisites
+
+(The following examples are run on Ubuntu 20.04 LTS.)
+
+First the two required libraries need to be installed:
+
+```sh
+$ sudo apt-get install libgtk-3-dev
+$ sudo apt-get install libwebkit2gtk-4.0-dev
+```
+
+### Installing Meson build system with pip
+
+Meson is available in the [Python Package Index](https://pypi.python.org/pypi/meson/) and can be installed with `pip3 install --user meson`. This does not require any special privileges. This will install the package in `~/.local/`, so you will have to add `~/.local/bin` to your `PATH`.
+
+```sh
+$ sudo apt-get install python3 python3-pip python3-setuptools \
+                       python3-wheel ninja-build
+$ pip3 install --user meson
+```
```

Comment:
While the content is fine, I think it may be better to leave it out of the tutorial, because each Linux distribution may have different instructions. If we wanted to add instructions for other systems in the future then we will end with a long part here which is not directly related to teaching how to write a small program that uses WebKitGTK.

Probably it's better to leave it out and assume people can figure out how to install the development packages in their distribution of choice 🤔️